script/Ciel/Day 13/13_01_02_CIEL13_7F.txt

```diff
@@ -658,32 +758,36 @@ I think that a time will come when you'll learn to enjoy life, so please don't c
 [sha:a60977ed480cb2809beb33809dc32d6b5b643a1b]{
 -- Page 455, Offset 36579.
 -- 「…………そう、でしょうか。
-"......Is that so?
+"I'm... no so sure.
 }
 [sha:ea085da40821bd590d598ac715ce06276618c297]{
 -- Page 455, Offset 36580.
 -- それは、夜に虹がかかるような話だと、思います」
-I think... that'd be like seeing a rainbow at night."
+// 15A chap title is currently "Rainbow In The Night", which is slightly more
+// awkward in this sentence than "Rainbow At Night" would be if we want
+// to just change the chp title now
+I think... that'd be like coming across a rainbow in the night."
```

Comment:
Rainbow In The Night is the official title of the chapter as found in the Japanese version so 

😢 🔫